site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+_**EncryptionKmsKey**_  
+The ARN of the [AWS Key Management Service (KMS)](https://docs.aws.amazon.com/kms) key that's used to encrypt application logs.
+
+_**HealthCheckProtocol**_  
+The IP protocol that AWS App Runner uses to perform health checks for your service. Valid values are "HTTP" and "TCP" (the default).
+
+_**HealthCheckPath**_  
+The URL that health check requests are sent to. This parameter is valid only if `HealthCheckProtocol` is set to "HTTP".
+
+_**HealthCheckInterval**_  
+The time interval, in seconds, between health checks. The valid range of values if 1 through 20, and the default is 5.
+
+_**HealthCheckTimeout**_  
+The time, in seconds, to wait for the response before deciding that a health check failed. The valid range of values if 1 through 20, and the default is 2.
+
+_**HealthCheckHealthyThreshold**_  
+The number of consecutive checks that must succeed before AWS App Runner decides that the service is healthy. The valid range of values if 1 through 20, and the default is 3.
+
+_**HealthCheckUnhealthyThreshold**_  
+The number of consecutive checks that must fail before AWS App Runner decides that the service is unhealthy. The valid range of values if 1 through 20, and the default is 3.
+
+_**VPCConnector**_  
+The VPC connector that associates your App Runner service with a custom Amazon VPC. For more information about VPC connectors, see [Networking with App Runner](https://docs.aws.amazon.com/apprunner/latest/dg/network.html) in the [AWS App Runner Developer Guide](https://docs.aws.amazon.com/apprunner/latest/dg/what-is-apprunner.html).
+
+By default, the deployment tool will not create a new VPC connector, but expects the ID of an existing VPC connector in `VpcConnectorId`.  
+To create a new VPC connector, set `CreateNew` to `true` and provide the following information:
+
+* `VpcId`  
+The VPC ID that AWS App Runner should use when it associates your service with a custom Amazon VPC.
+* `Subnets`  
+A list of subnet IDs that AWS App Runner should use when it associates your service with a custom Amazon VPC. Specify IDs of subnets of a single Amazon VPC. AWS App Runner determines the Amazon VPC from the subnets you specify.
+* `SecurityGroups`  
+A list of IDs of security groups that AWS App Runner should use for access to AWS resources under the specified subnets. If not specified, AWS App Runner uses the default security group of the custom Amazon VPC. The default security group allows all outbound traffic.
+
+_**AppRunnerEnvironmentVariables**_  
+Environment properties for your application.
+
+## JSON syntax for Docker container image to Amazon ECR
+
+Use this JSON definition to push a Docker container image to [Amazon Elastic Container Registry (Amazon ECR)](https://docs.aws.amazon.com/ecr) (`RecipeId` set to "PushContainerImageEcr").
+
+```
+{
+    "AWSProfile": "string",
+    "AWSRegion": "string",
+    "StackName": "string",

```

Comment:
Does this parameter apply to this recipe?

Reply:
I don't think it will, but I will defer the question to Malhar